api.ts

```diff
@@ -193,7 +199,7 @@ export interface SyncTableDef<
   /** See {@link SyncTableOptions.name} */
   name: string;
   /** See {@link SyncTableOptions.description} */
-  description?: string;
+  description?: DescriptionToken[];
```

Comment:
As you noted, this and the other fields will probably have to stay `string | DescriptionToken[]` forever, because existing packs will have this as `string`. Even if we disallow old packs at runtime at some point, if we are ever iterating over older pack versions in some workflow, we'd want the typing to be accurate to their contents.

Reply:
Done.